Source/objects.cpp

```diff
@@ -5195,8 +5180,11 @@ void ObjChangeMap(int x1, int y1, int x2, int y2)
 		ObjL2Special(2 * x1 + 16, 2 * y1 + 16, 2 * x2 + 17, 2 * y2 + 17);
 		AddL2Objs(2 * x1 + 16, 2 * y1 + 16, 2 * x2 + 17, 2 * y2 + 17);
 	}
+	if (leveltype == DTYPE_CAVES) {
+		AddL3Objs(2 * x1 + 16, 2 * y1 + 16, 2 * x2 + 17, 2 * y2 + 17);

```

Comment:
This doesn't change behavior, cause in vanilla caves never uses this function.
Is my assumption correct?

Reply:
Poisoned water is the only custom vanilla caves map. And it doesn't change it layout. The portion of code is also only relevant if the map changes to add fence doors to an area, not so likely, but good to have just in case :)